### DISCUSSION TITLE
Configuration question

### DISCUSSION THREAD

**Discussion by hranskw122** [gickowtf/pixoo-homeassistant#45] [Q&A] [answered]
Hi,

Great integration! 

I managed to configure the pages in the Configuration page of the device and the send a page using the service call.  But how can I define the pages the configuration file?

Thanks,
Norbert

**Comment by SB2DD** [comment, marked as answer]
Edit: I'm so sorry, I misread your message. It's not possible anymore to use the configuration file to setup the integration. Please use the configuration page instead. 

~~The configuration file isn't used anymore. You have to set it up in the device configuration page (See screenshot below). In there, you can define a list of pages. There is a great example in the top of the README.~~

~~You can access this page by going in your Settings->Device and services->Divoom Pixoo 64
<img width="998" alt="image" src="https://github.com/gickowtf/pixoo-homeassistant/assets/45701824/f41edafb-086e-4e93-8fe0-416833d77d3e">
(There, you would click the "Configure" (In my case configurer) button to be able to select the list of pages.)~~
